pkg/controller/v1alpha2/applicationconfiguration/render.go

```diff
@@ -619,6 +619,7 @@ func getTraitName(ac *v1alpha2.ApplicationConfiguration, componentName string,
 		for _, trait := range w.Traits {
 			if trait.Reference.APIVersion == apiVersion && trait.Reference.Kind == kind {
 				traitName = trait.Reference.Name
+				return traitName
```

Comment:
I think this PR is not necessary, and it will cause bug if we return here. because traitName here could be empty.


What's the most need here is a test case. 😄

Reply:
So add 'break' here is a good way.